checkstyle.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
```

Comment:
Just wanted to know. What does this file do ?

Reply:
It defines the code style rules which will be checked during the build / release. We applied this check for all of our Jenkins plugins to ensure consistent code style.

Check the file pom.xml , there is a plugin configuration which references this file.